source/blog/2017-11-05-vdsm-setup-networks-is-now-much-quicker.html.md

```diff
@@ -0,0 +1,40 @@
+---
+title: VDSM setupNetworks is now much quicker
+author: phoracek
+tags: ovirt, 4.2.0, network
+date: 2017-11-05 09:00:00 CET
+comments: true
+published: true
+---
+The easiest way to handle high number of networks in cluster is to attach them
+to a label. However, if there is too many of them, Engine could fail to setup
```

Comment:
please add the follow clause:

attach them to a label, **and then place that label on a NIC of the added host**. However, ...

Reply:
Done